pom.xml

```diff
@@ -1,34 +1,55 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<parent>
-		<artifactId>mnode-parent</artifactId>
-		<groupId>org.mnode</groupId>
-		<version>1.16</version>
-	</parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.mnode.ical4j</groupId>
+	<groupId>org.bedework</groupId>
 	<artifactId>ical4j-vcard</artifactId>
```

Comment:
You should change the artifactId here to match the repository name (bw-ical4j-vcard).